src/core/lib/experiments/config.cc

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "src/core/lib/experiments/config.h"
+
+#include "absl/strings/str_split.h"
+
+#include <grpc/support/log.h>
+
+#include "src/core/lib/experiments/config.h"
+#include "src/core/lib/experiments/experiments.h"
+#include "src/core/lib/gprpp/global_config.h"
+#include "src/core/lib/gprpp/no_destruct.h"
+
+GPR_GLOBAL_CONFIG_DEFINE_STRING(
+    grpc_experiments, "",
+    "List of grpc experiments to enable (or with a '-' prefix to disable).");
```

Comment:
Nit: re-word to "comma-delimited string of grpc experiments..."

Reply:
also, this string should always default to include any experiments that default to true, correct? Looks like this is the default behavior anyway if the string is empty, but might be good to make sure we explicitly include all the experiments enabled in the env var by default (can be modified externally).

Reply:
Can you explain the advantage in doing that, I'm not seeing it.

Reply:
I was thinking if we are transparently enabling the experiments even if the env var is empty, users might not be aware of the experiments, but it would be much clearer to them if the default enabled experiments showed up in env vars / flags. If there's an easy way for users (and us when debugging) to know which experiments are enabled by default as is, that works too; I may be missing something.

Reply:
I might do something analogous to trace then, and have a 'list_experiments' experiment.
My worry is when we start having internal experiments and wanting to merge the lists and what not things are going to get crazy with the help text.
Eventually we'll codegen a .md file too I guess.

Reply:
(we'll also have some monitoring on what's enabled)

Reply:
(updated with the logs we discussed offline)